its/src/test/java/com/sonar/it/scanner/msbuild/sonarcloud/CloudUtils.java

```diff
@@ -46,11 +46,17 @@ public static void waitForTaskProcessing(String logs) {
 
       await()
         .pollInterval(Duration.ofSeconds(5))
-        .atMost(Duration.ofMillis(Timeout.FIVE_MINUTES.miliseconds))
+        .atMost(Duration.ofMillis(Timeout.TWO_MINUTES.miliseconds))
```

Comment:
Bringing back to the old value which was:
```suggestion
        .atMost(Duration.ofSeconds(120))
```